src/components/primitives/Select/Select.tsx

```diff
@@ -154,6 +154,8 @@ const Select = (
     'shadow',
     'opacity',
   ]);
+  
+  const isSafari = Platform.OS === 'web' && window.safari
```

Comment:
```suggestion
  const isSafari = Platform.OS === 'web' && navigator.userAgent.match(/Version\/[\d.]+.*Safari/)
```

Actually, in iPhone, the `safari` property is not set.